README.md

```diff
@@ -27,7 +27,10 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 
 Write your code here:
 ```ruby
-# code here
+def offer_rose (person)
+  puts "Would you take this rose, `person`, in exchange for giving an old beggar woman shelter from the bitter cold?"
```

Comment:
String interpolation works differently in Ruby. Instead of wrapping person in backticks, you need to do this: `#{person}`